app/src/main/java/com/vultisig/wallet/ui/models/swap/SwapFormViewModel.kt

```diff
@@ -290,14 +308,20 @@ internal class SwapFormViewModel @Inject constructor(
                         UiText.StringResource(R.string.send_error_no_token)
                     )
 
-                if (selectedSrcBalance < srcAmountInt
-                    || nativeTokenValue < (estimatedNetworkFeeTokenValue.value?.value
-                        ?: BigInteger.ZERO) + swapFee
+                if (
+                    selectedSrcBalance < srcAmountInt + nonNativeswapFee
```

Comment:
This check is still wrong, `nativeSwapFee` / `nonNativeSwapFee` should not interfere with the amount that can be used for swap. the inbound amount should be only consider 

`nativeSwapFee` / `nonNativeSwapFee` means how much will be deducted from the inbound token for the swap. 

which means , if you use `nativeSwapFee`  / `nonNativeSwapFee` in any calculation , it will be wrong

The amount that can be used for swap should only be considered  by user's input and gas fee for the outbound tx